src/main/java/com/mraof/minestuck/client/renderer/entity/GristRenderer.java

```diff
@@ -15,41 +16,63 @@
 
 public class GristRenderer extends EntityRenderer<GristEntity>
 {
-	
+	static final double shakeScale = 1.0 / 256.0;
```

Comment:
Constants like this one use SCREAMING_SNAKE_CASE.

Reply:
fixed